lib/BasicTypes.cpp

```diff
@@ -54,7 +54,7 @@ int AFactionMember::getAttack(bool ranged) const
 {
 	const std::string cachingStr = "type_PRIMARY_SKILLs_ATTACK";
 
-	static const auto selector = Selector::typeSubtype(BonusType::PRIMARY_SKILL, static_cast<int>(PrimarySkill::ATTACK));
+	static const auto selector = Selector::typeSubtype(BonusType::PRIMARY_SKILL, TBonusSubtype(PrimarySkill::ATTACK));
```

Comment:
`TbonusSubtype` -> `BonusSubtype`?

Reply:
BonusSubtypeID is probably more in line with our naming scheme. Will rename. Same with TBonusSourceID -> BonusSourceID